From 4b0616ddd9be0474677aac9b147fc6a1aa42810e Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Mon, 21 Jul 2014 17:36:27 +0800
Subject: [PATCH 221/255] arm: dts: ls1021a: change sysclk compatible and
 remove the tbi node

The sysclk can be probed by "fixed-clock" compatible,
use "fsl,qoriq-sysclk-2.0" to let uboot find and add
clock frequency property.
Here also remove the tbi node which will be added to
boards level dts.

Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
[Kevin: Original patch taken from
LS1021A-SDK-V1.1-ARM-SOURCE-20140815-yocto.iso]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 0b4b686..a57f5f8 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -228,7 +228,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			sysclk: sysclk {
-				compatible = "fsl,sys-clock";
+				compatible = "fsl,qoriq-sysclk-2.0", "fixed-clock";
 				#clock-cells = <0>;
 				clock-frequency = <100000000>;
 				clock-output-names = "sysclk";
@@ -647,10 +647,6 @@
 			device_type = "mdio";
 			compatible = "gianfar";
 			reg = <0x0 0x2d24000 0x0 0x4000>;
-			tbi0: tbi-phy@8 {
-				reg = <0x8>;
-				device_type = "tbi-phy";
-			};
 		};
 
 		enet0: ethernet@2d10000 {
-- 
2.0.2

